MapboxSearch/base/build.gradle

```diff
@@ -88,7 +88,7 @@ dependencies {
     testImplementation "org.junit.jupiter:junit-jupiter-engine:$junit_5_version"
     testImplementation project(":common-tests")
 
-    androidTestImplementation "com.mapbox.common:okhttp:$common_sdk_version"
+    androidTestImplementation "com.squareup.okhttp3:okhttp:4.12.0"
```

Comment:
It's better to extract `4.12.0` to constants in `dependencies.gradle`

Reply:
It requires introducing a new file, I left it the way it is right now to minimize changes (there are a lot of them already).